pkg/kudoctl/cmd/init.go

```diff
@@ -32,6 +31,9 @@ version. You can specify an alternative image with '--kudo-image' which is the f
 or '--version' which will replace the version designation on the standard image.
 
 To dump a manifest containing the KUDO deployment YAML, combine the '--dry-run' and '--output=yaml' flags.
+
+Running 'kudo init' on server-side is idempotent - it skips manifests alredy applied to the cluster in previous runs
+and finishes with success if KUDO is already installed.
```

Comment:
I'm not sure if necessary or not... but an important part of this message to me is that if the k8s objects exists... the create is skipped... there is no update.   If `kudo init` was applied with version `0.7.0` followed by a future `0.8.0` there is no update to the objects.   It would have to be uninstalled reinstalled currently.

Reply:
hmm, interesting, I actually did not know that so maybe that should be in the documentation as well 🤔